src/codec/binflux.rs

```diff
@@ -30,9 +30,11 @@ const TYPE_FALSE: u8 = 4;
 pub struct BInflux {}
 
 impl BInflux {
+    #[allow(clippy::map_err_ignore)]
     pub fn encode(v: &simd_json::BorrowedValue) -> Result<Vec<u8>> {
         fn write_str<W: Write>(w: &mut W, s: &str) -> Result<()> {
             w.write_u16::<BigEndian>(
+                #[allow(clippy::map_err_ignore)]
```

Comment:
I don't think we need to allow this, including the error into the error will give a more descriptive error.